app/views/games/summary.html.haml

```diff
@@ -2,9 +2,9 @@
   Game Summary
 
 %h3.muted
-  Top 5 Teams
+  Teams Summary
 %br/
-= render partial: "games/divisions", locals: { divisions: @game.divisions, active_division: @active_division, only_top_five: true }
+= render partial: "games/divisions", locals: { divisions: @game.divisions, active_division: @active_division, only_top_five: false}
```

Comment:
`only_top_five` can be completely removed from everywhere in the application since it is only used here.